sonic-ledd/scripts/ledd

```diff
@@ -183,21 +69,17 @@ def main():
                 print 'ledd version ' + VERSION
                 sys.exit(0)
 
-    # Register our signal handlers
-    signal.signal(signal.SIGHUP, signal_handler)
-    signal.signal(signal.SIGINT, signal_handler)
-    signal.signal(signal.SIGTERM, signal_handler)
-
     # Load platform-specific LedControl module
-    led_control = load_platform_led_control_module()
-    if led_control is None:
-        sys.exit(0)
+    led_control = ledd_util.load_platform_util(LED_MODULE_NAME, LED_CLASS_NAME)
+    if not led_control:
+        ledd_util.log_error("failed to load ledutil")
+        sys.exit(1)
 
     # Open a handle to the Application database
     appl_db = swsscommon.DBConnector(swsscommon.APPL_DB,
-                                     REDIS_HOSTNAME,
-                                     REDIS_PORT,
-                                     REDIS_TIMEOUT_USECS)
+                                     ledd_util.redis_hostname,
```

Comment:
DB connector creation can also go to base class. 3 of 4 parameters belong to base
here can be only:
appl_db = my_daemon.DBConnect(APPL_DB)